src/pages/team.tsx

```diff
@@ -1,93 +1,97 @@
+    bio: "React Native developer with native Android and iOS experience. Always curious and eager to try new technologies. Often tinkers with electronics or rides a bike.",
   },
   {
-    name: 'Łukasz',
-    imageUrl: require('../assets/team/lukasz.jpg'),
-    bio:
-      'React Native developer with web experience. Enthusiast of functional programming. Always on the lookout for performance gains, never stops asking why. Can be found skating down the streets.',
+    name: "Łukasz",
+    photo: "lukasz.jpg",
+    bio: "React Native developer with web experience. Enthusiast of functional programming. Always on the lookout for performance gains, never stops asking why. Can be found skating down the streets.",
   },
   {
-    name: 'Błażej',
-    imageUrl: require('../assets/team/blazej.jpg'),
-    bio:
-      'Mobile apps developer. He fell in love with React Native. Self-improver, golf enthusiast. In his house you can always count on being welcomed with a good Scotch.',
+    name: "Błażej",
+    photo: "blazej.jpg",
+    bio: "Mobile apps developer. He fell in love with React Native. Self-improver, golf enthusiast. In his house you can always count on being welcomed with a good Scotch.",
   },
   {
-    name: 'This could be you',
-    imageUrl: require('../assets/team/open-positions.jpg'),
-    bio:
-      'Ready to take the next step in your career? Drop us a message to see if you could join our team.',
+    name: "This could be you",
+    photo: "open-positions.jpg",
+    bio: "Ready to take the next step in your career? Drop us a message to see if you could join our team.",
   },
-]
+];
+
+const title = "Team";
+const description =
+  "Technology is never enough to solve a problem. You'll need a team of experienced and caring developers as well. How about ours?";
 
 class TeamPage extends Component {
   render() {
     const {
       versatilityIcon,
       reliabilityIcon,
       diversityIcon,
-      outsourcingIcon,
-    } = this.props.data
+      testimonialImage,
+      teamImages,
+    } = this.props.data;
+
+    const teamMembersWithImages = teamMembers.map((it) => {
+      const image = teamImages.nodes.find(
+        (file) => file.relativePath === it.photo
+      );
+      return {
+        ...it,
+        image,
+      };
+    });
 
     return (
-      <Layout
-        headerTitle="Team"
-        headerSub="Technology is never enough to solve a problem. You'll need a team of experienced and caring developers as well. How about ours?"
-      >
+      <Layout headerTitle={title} headerSub={description}>
         <div className="content-page">
           <div className="big-part-yellow">
             <div className="content members-box">
-              {teamMembers.map((member) => (
+              {teamMembersWithImages.map((member) => (
                 <TeamMember {...member} key={member.name} />
               ))}
             </div>
             <div className="mobile-only team-swiper">
               <Swiper
```

Comment:
On "Team" page on smartphone I see only Patryk.

In fact it shows a swiper, but not centered and without any arrows indicating it's possible to swipe it. 

<img width="512" alt="Screenshot 2023-04-19 at 10 02 38" src="https://user-images.githubusercontent.com/2790570/233009717-0cdcade7-5357-4574-b72a-bcdd25b03a72.png">

Reply:
Fixed 👍